src/types/api.ts

```diff
@@ -15,3 +15,46 @@ export type APIPushResponse = {
   meta: APIPushMeta;
   results: Changeset[];
 };
+
+export type JobInformation = {
+  build_architecture: string;
+  build_os: string;
+  build_platform: string;
+  build_platform_id: number;
+  build_system_type: 'taskcluster';
+  end_timestamp: number;
+  failure_classification_id: number; // probably an enum
+  id: number;
+  job_group_description: string;
+  job_group_id: number;
```

Comment:
Mmm I'm curious whether job_group_id could be our decisionTaskId after all? We can look at it later.